plugins/dashboard_pi/src/baro_history.cpp

```diff
@@ -284,7 +284,10 @@ void DashboardInstrument_BaroHistory::DrawForeground(wxGCDC* dc)
   m_ratioW = double(m_DrawAreaRect.width) / (BARO_RECORD_COUNT-1);
  // dc->DrawText(wxString::Format(_(" Max %.1f Min %.1f since %02d:%02d  Overall Max %.1f Min %.1f "),m_MaxPress,m_MinPress,hour,min,m_TotalMaxPress,m_TotalMinPress), m_LeftLegend+3+2+degw, m_TopLineHeight-degh+5);
  // Cant get the min sice to work...
- dc->DrawText(wxString::Format(_(" Max %.1f since %02d:%02d  Overall Max %.1f Min %.1f "),m_MaxPress,hour,min,m_TotalMaxPress,m_TotalMinPress), m_LeftLegend+3+2+degw, m_TopLineHeight-degh+5);
+  wxString sMax = _( "Max" ), sSince = _( "since" ), sOverall = _( "Overall" ), sMin = _( "Min" );
```

Comment:
This is a very, very unfortunate idea. What is for example "Min"? Minimum? Minute? Is it always the same?
Breaking the strings to be translated to single words, just to "protect" the translators from themselves is a bad practice and we should get rid of it in the codebase, not introduce more.
The way to resolve this kind of stuff is to actually start using https://github.com/OpenCPN/OpenCPN/pull/387 and fix the translations when they are the root cause of the problem.